src/providers/ibmcloud/mod.rs

```diff
@@ -196,4 +249,116 @@ foo:      ba:r
             Some(&"test_instance-vpc-gen2".to_string())
         );
     }
+
+    #[test]
+    fn test_fetch_ssh_keys() {
+        let vendordata = concat!(
```

Comment:
I think you are manually escaping a lot of things in here. You may consider using a [raw string](https://doc.rust-lang.org/reference/tokens.html#raw-string-literals) instead:
```rust
let vendordata = 
r#"your
raw
"quotes"
in
here!"#;
```

or directly put this content into a fixture file (under `tests/fixtures/`).

Reply:
I moved this to a fixture file in the next commit.